docs/user/guides/hosting-integrations.rst

```diff
@@ -0,0 +1,97 @@
+Hosting Integrations
+====================
+
+Read the Docs performs some actions by default when building documentation for its users to.
```

Comment:
It's like something is off with this sentence?

But a good way to start it, could be like writing to the reader and the author and reviewers: "In this section, you can learn about <...>".

Reply:
I updated this a little to mention this as the first sentence. I think it's better now 👍🏼

Reply:
I still didn't entirely understand it, maybe it's just a redundant "to" at the end.

Here's what I mean by introducing the article in a way that tells both the author and the reader what the scope and intention of the article is... I think it's nice to start with, it can maybe be trimmed later.

```suggestion
Read the Docs performs a set of default actions to build and publish your documentation. In this article, we explain more about these actions, how they work, what they produce, how they relate. Ultimately we give you pointers on how to configure them.
```

Reply:
Great change :+1: 

I see also that "explain" is used, which is really good. At the end of content reorganizing, the idea of having a real reference for features will come up (so let's hope we are there in 1 month :crossed_fingers: ). In that regards, this overview and explanation will be valuable because it means that the feature reference can simply link to the explanation article.